clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/ImageRepositoryProvider.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.provider.view;
+
+public interface ImageRepositoryProvider {
+  String getRepositoryName();
+
+  boolean handles(String url);
+
+  Object findImage(String url);
```

Comment:
I'm fine with typing it as an Object, but I can imagine this being questioned in upstream code review.  Up to you whether you type it into something or not

Reply:
I just kept the type that it originally returned.

If we can switch the type, sure I will make an object.

Reply:
Done.

Returns a `List<EcsDockerImage>`.

Chose to go with `Ecs` instead of `Ecr` since `Ecs` is more generic, and is intended for the ECS clouddriver use.